### PR TITLE
Adb/config/tests

### PR DESCRIPTION
**Pull request by adbarc92**
* Adds some initial query data for testing purposes - SKIP THESE FILES
* Rearranges File Structure
* Adds Cheese

### CODE REVIEW

File: src/db/createTables.sql [hunk 1]
```diff
@@ -53,11 +55,11 @@ CREATE TABLE IF NOT EXISTS qa.answers (
   id INT PRIMARY KEY,
   question_id INT NULL DEFAULT NULL,
   body TEXT NULL DEFAULT NULL,
-  date_written BIGINT NULL DEFAULT NULL,
+  answer_date BIGINT NULL DEFAULT NULL,
   answerer_name TEXT NULL DEFAULT NULL,
 	answerer_email TEXT NULL DEFAULT NULL,
 	reported BOOLEAN NULL DEFAULT NULL,
```
Comment (alishaedington) [inline src/db/createTables.sql:61]: why are these indented? any specific reason?

File: src/index.ts [hunk 2]
```diff
@@ -1,47 +1,115 @@
...
 /* MIDDLEWARE */
 
 app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
 
 /* Questions List */
-app.get('/qa/questions', (req: Request, res: Response) => {
-  console.log('url:', req.url);
-  res.status(200).send('pizza');
+app.get('/qa/questions', async (req: Request, res: Response) => {
```
Comment (alishaedington) [inline src/index.ts:20]: all of these are in the same file as your server connection? are you gonna seperate eventually?

File: src/utils.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  return null;
+};
+
+export const composeQuery = (
+  baseQuery: string,
+  count: number,
+  page: number,
+): string => {
+  return `${baseQuery} LIMIT ${count} OFFSET ${(page - 1) * count};`;
+};
```
Comment (alishaedington) [inline src/utils.ts:19]: What is this doing?

**Review by alishaedington — APPROVED** [review]
a few comments, i'm sure your code is 92% different now.

File: src/db/createTables.sql (hunk 1 above)
Comment (adbarc92) [reply src/db/createTables.sql:61]: that is not how they look in my IDE. Strange.

**Pull request merged**